src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier;
+    private final Random random;
+
+    public Lottery() {
+        this.colorSupplier = new ColorSupplier();
+        this.random = new Random();
+    }
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor();
+        int randomInt = random.nextInt(101);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/3ed5ce59-10d1-4dfe-b6ba-53a6cb239a79)

Reply:
int MAX_BALL_NUMBER has been added for clarity of the code.